site/gatsby-site/src/global.css

```diff
@@ -78,7 +78,7 @@ a {
   text-decoration: none;
 }
 a:hover {
-  text-decoration: none;
+  text-decoration: underline;
```

Comment:
Thanks! Can you also add `text-underline-offset: 4px` so it has more space to breath? After that I'm good to merge.